clone.js

```diff
@@ -0,0 +1,104 @@
+function initializeWebcomponent(options) {
+	copyFolderSync(process.cwd(), options.path, [".git", "node_modules", "util", "README.md", "clone.sh", "package-lock.json"]);
+	replaceDescriptionInReadMe(path.resolve(options.path, "README.md.template"), options.description);
+	replaceDescriptionInPackageJson(path.resolve(options.path, "package.json"), options.description);
+	replaceInFile(path.resolve(options.path, "package.json"), options.naam);
+	replaceInFile(path.resolve(options.path, "src/vl-blueprint.src.js"), options.naam);
+	replaceInFile(path.resolve(options.path, "src/style.scss"), options.naam);
+	replaceInFile(path.resolve(options.path, "README.md.template"), options.naam);
+	replaceInFile(path.resolve(options.path, "index.js"), options.naam);
+	replaceInFile(path.resolve(options.path, "src/index.js"), options.naam);
+
+	replaceInFile(path.resolve(options.path, "bamboo-specs/bamboo.yml"), options.naam);
+	replaceInFile(path.resolve(options.path, "demo/vl-blueprint.html"), options.naam);
+	replaceInFile(path.resolve(options.path, "test/e2e/components/vl-blueprint.js"), options.naam);
+	replaceInFile(path.resolve(options.path, "test/e2e/pages/vl-blueprint.page.js"), options.naam);
+	replaceInFile(path.resolve(options.path, "test/e2e/blueprint.test.js"), options.naam);
+	replaceInFile(path.resolve(options.path, "test/unit/vl-blueprint.test.html"), options.naam);
+
+	rename(path.resolve(options.path, "src/vl-blueprint.src.js"), path.resolve(options.path, `src/vl-${options.naam}.src.js`));
+	rename(path.resolve(options.path, "README.md.template"), path.resolve(options.path, "README.md"));
+
+	rename(path.resolve(options.path, "demo/vl-blueprint.html"), path.resolve(options.path, `demo/vl-${options.naam}.html`));
+	rename(path.resolve(options.path, "test/e2e/components/vl-blueprint.js"), path.resolve(options.path, `test/e2e/components/vl-${options.naam}.js`));
+	rename(path.resolve(options.path, "test/e2e/pages/vl-blueprint.page.js"), path.resolve(options.path, `test/e2e/pages/vl-${options.naam}.page.js`));
+	rename(path.resolve(options.path, "test/e2e/blueprint.test.js"), path.resolve(options.path, `test/e2e/${options.naam}.test.js`));
+	rename(path.resolve(options.path, "test/unit/vl-blueprint.test.html"), path.resolve(options.path, `test/unit/vl-${options.naam}.test.html`));
+}
+
+function replaceDescriptionInReadMe(path, description) {
+	replace(path, "@description@", description);
+}
+
+function replaceDescriptionInPackageJson(path, description) {
+	const data = fs.readFileSync(path, 'utf8');
+	var packageJsonData = JSON.parse(data);
+	packageJsonData.description = description;
+	const result = JSON.stringify(packageJsonData, null, '\t');
+	fs.writeFileSync(path, result, 'utf8');
+}
+
+function replaceInFile(path, naam) {
+	const naamLowercase = naam;
+	const naamCamelcase = naam.replace(/(^|[\s-])\S/g, function (match) {
+	    return match.toUpperCase();
+	}).replace(/-/g, "");
+	const naamUppercase = naam.toUpperCase().replace(/-/g, "");
```

Comment:
"naamUppercaseEnZonderDashes" duid iets beter aan wat het eigenlijk is?
Waarom eigenlijk naamUppercase wel zonder dashes, maar de lowercase variant niet?

Reply:
De naamUppercase wordt gebruikt voor de JS class naam.